docs/features/datastores/overview.md

```diff
@@ -0,0 +1,31 @@
+# Demystifying Datastorses
+
+## What are Datastores
+
+Datastores are high-performance, low-latency, versioned, and scalable datasets.
+
+## What files can be accepted
+Datastores can be created from any file type
+
+## Why use Datastores
+Datastores are zero cost datasets backed by cloud storage. **Add additional content as to why this is better than local directory storage etc**.
+
+### Capabilities
+Datastores today have 3 main capabilities:
+1. Can be created from private S3 buckets in both Grid Cloud and Bring Your Own Cloud environments
```

Comment:
Unclear if we will support private s3 for grid cloud

Reply:
It is something that was said should work and is something that there is a linear ticket for. Whether it is a documentation thing or a broken feature is something that needs to be discerned.